Assets/SteamVR/Scripts/SteamVR.cs

```diff
@@ -19,7 +19,7 @@ public static bool enabled
 	{
 		get
 		{
-			if (!UnityEngine.VR.VRSettings.enabled)
+			if (!UnityEngine.XR.XRSettings.enabled)

```

Comment:
I'm not sure how VR's are changed to XR's, but they seem to cause the merge conflicts. Same in VRWorks.cs and SteamVR_Camera.cs

Reply:
Did not change that one